modules/ROOT/pages/php.adoc

```diff
@@ -198,120 +65,13 @@ Advanced Neo4j users will have full control over the client and what Cypher gets
 [#neoeloquent-lib]
 == NeoEloquent for Laravel
 
-Neo4j Graph Eloquent Driver for Laravel 4
+Neo4j Graph Eloquent Driver for Laravel
 
 [cols="1,4"]
```

Comment:
this still uses the GraphAware driver or?

Reply:
true, I can remove it for now